client/src/components/Draftail/blocks/EmbedBlock.js

```diff
@@ -4,16 +4,20 @@ import React from 'react';
 import { STRINGS } from '../../../config/wagtailConfig';
 
 import MediaBlock from '../blocks/MediaBlock';
+import { shortenLabel } from '../decorators/TooltipEntity'
 
 /**
  * Editor block to display media and edit content.
  */
 const EmbedBlock = props => {
   const { entity, onRemoveEntity } = props.blockProps;
-  const { url, title, thumbnail } = entity.getData();
+  const { url, title, thumbnail, providerName, authorName } = entity.getData();
+  // Fallback text is used when there is no image available for the embed.
+  // In those cases, it seems like author and provider name will always be present.
+  const fallbackText = `${shortenLabel(authorName)}\n${shortenLabel(providerName)}`;
 
   return (
-    <MediaBlock {...props} src={thumbnail} alt="">
+    <MediaBlock {...props} src={thumbnail} alt="" fallbackText={fallbackText}>
```

Comment:
Might need to update the existing test case(s) to check that this prop is set as expected.